test/testdata/desugar/forward_args.rb.desugar-tree.exp

```diff
@@ -0,0 +1,29 @@
+class <emptyTree><<C <root>>> < (::<todo sym>)
+  class <emptyTree>::<C Foo><<C <todo sym>>> < (::<todo sym>)
+    def bar<<C <todo sym>>>(a, b, k1:, k2:, &block)
+      <emptyTree>
+    end
+
+    def foo<<C <todo sym>>>(*args, *kargs:, &block)
+      ::<Magic>.<call-with-splat-and-block>(<self>, :bar, ::T.unsafe(args).to_a().concat([kargs.to_hash()]), block)
+    end
+
+    def foo2<<C <todo sym>>>(*args, *kargs:, &block)
+      ::<Magic>.<call-with-splat-and-block>(<self>, :bar, <emptyTree>::<C T>.unsafe(args).to_a().concat([kargs.to_hash()]), nil, block)
```

Comment:
Not sure what the `nil` passed here is about. Is this the `convertedBlock`?

Reply:
This is part of the keyword args rework: for `call-with-splat-and-block` we pass the positional args and keyword args as separate arrays to the same intrinsic now.